changelog/upgrade-notes.md

```diff
@@ -1,5 +1,10 @@
 ## Upgrade notes
 
+### Next release
```

Comment:
I think you should explicitly mark that as a breaking change

Reply:
Isn't the upgrade notes all about breaking changes? I don't see any previous changes marked explicitly as breaking there.
What do you have in mind?

Reply:
For version 8 the breaking changes were put in separate section: https://github.com/openlayers/openlayers/blob/main/changelog/upgrade-notes.md#backwards-incompatible-changes

This is what I had in mind